examples/vision/attention_mll_classification.py

```diff
@@ -0,0 +1,541 @@
+    count = 0
+
+    # Take out the filter for specific class.
+    filter_class = np.where(y == pos_class)[0]
+
+    # Assign new variables consisting of this class.
+    x_pos_class = x[filter_class]
+    y_pos_class = y[filter_class]
+
+    # From overall data, remove this class.
+    x_neg_classes = np.delete(x, filter_class, 0)
+    y_neg_classes = np.delete(y, filter_class, 0)
+
+    # Merge both inputs and labels to each another.
+    x_merged = np.concatenate([x_pos_class, x_neg_classes], axis=0)
+    y_merged = np.concatenate([y_pos_class, y_neg_classes], axis=0)
+
+    # Data are ordered in such a way: [pos_class... neg_classes]
+    # Shuffle the data randomly.
+    order = np.arange(len(x_merged))
+    np.random.shuffle(order)
+    x = x_merged[order]
+    y = y_merged[order]
+
+    for _ in range(bag_count):
+
+        # Pick a fixed size random subset of samples.
+        index = np.random.choice(x.shape[0], instance_count, replace=False)
+        images = x[index]
+        labels = y[index]
+
+        # By default, all bags are labelled as 0.
+        label = 0
+
+        # Check if there is atleast a positive class in the bag.
+        if pos_class in labels:
+
+            # Positive bag will be labelled as 1.
+            label = 1
+
+            # Increment count by 1.
+            count += 1
+
+        bags.append(images)
+        bag_labels.append(np.array([label]))
+
+    print(
+        "Positive bags: " + str(count),
+        "Negative bags: " + str(bag_count - count),
+    )
+    return (list(np.swapaxes(bags, 0, 1)), np.array(bag_labels))
+
+
+# Load data.
+(x, y), (x_val, y_val) = DATASET["mnist"].load_data()
+
+# Create training data.
+train_data, train_labels = create_bags(x, y, POS_CLASS, BAG_COUNT, BAG_SIZE)
+
+# Create validation data.
+val_data, val_labels = create_bags(x_val, y_val, POS_CLASS, VAL_BAG_COUNT, BAG_SIZE)
+
+"""
+# Create and Train Neural Networks.
+In this section we will build the attention layer, write some function for training and visualizing, build and train the entire model.
+
+## Attention Operator Implementation
+
+The output size of this layer is decided by the size of a single bag.
+
+The attention mechanism uses a weighted average of instances in a bag, in which the sum of the weights must equal to 1 (invariant of the bag size).
+
+The weight matrices (parameters) are **w** and **V**. To include positive and negative values, hyperbolic tangent element-wise non-linearity is utilized.
+
+**Gated attention mechanism** can be used to deal with complex relations. Another weight matrix, **U**, is added to the computation.
+Sigmoid non-linearity is used to overcome approximately linear behavior for *x* ∈ [−1, 1] by hpyerbolic tangent non-linearity.
+"""
+
+
+class AttentionLayer(layers.Layer):
+    def __init__(
+        self,
+        L_dim,
+        output_dim,
+        kernel_initializer="glorot_uniform",
+        kernel_regularizer=None,
+        use_bias=True,
```

Comment:
Are the 3 args above really important to configure? Keep the code minimal

Reply:
`use_bias` was removed.